mopidy/commands.py

```diff
@@ -38,7 +38,8 @@ def config_override_type(value):
 
 
 class _ParserError(Exception):
-    pass
+    def __init__(self, message):
+        self.message = message
```

Comment:
Another way of fixing this warning is to subclass `mopidy.exceptions.MopidyException`, which reimplements the message field that was deprecated in Python 2.6.